pkgs/development/tools/database/pgcli/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pythonPackages, fetchFromGitHub }:
+
+pythonPackages.buildPythonPackage rec {
```

Comment:
:+1: for moving this application out of `pythonPackages`. Please do use `buildPythonApplication`.